scripts/verify-release-artifacts.sh

```diff
@@ -35,10 +39,16 @@ if [ $# -eq 0 ]; then
     exit 1
 fi
 
+if [ ! -d "$release_dir" ]; then
+    echo "$0: Release directory not found in $release_dir"
+    exit 1
+fi
+
 release_tag=$1
 # Strip 'v' from release tag.
 release_version=${release_tag/v/} 
 
+cd $release_dir
```

Comment:
nit: we should use `pushd` here and `popd` at the end of the file so we don't mess with a user's working directory after executing this script.